src/Utils.h

```diff
@@ -149,20 +150,21 @@ messageDescription(const QString &username = QString(),
             return QCoreApplication::translate("message-description sent:",
                                                "%1 sent a notification")
               .arg(username);
-    } else if (std::is_same<T, Text>::value) {
+    } else if (std::is_same<T, Text>::value || std::is_same<T, Unknown>::value) {
         if (isLocal)
             return QCoreApplication::translate("message-description sent:", "You: %1").arg(body);
         else
             return QCoreApplication::translate("message-description sent:", "%1: %2")
               .arg(username, body);
-    } else if (std::is_same<T, Confetti>::value) {
+    } else if (std::is_same<T, ElementEffect>::value) {
         if (body.isEmpty()) {
+            // TODO: what is the best way to handle this?
             if (isLocal)
                 return QCoreApplication::translate("message-description sent:",
-                                                   "You sent some confetti");
+                                                   "You sent a special effect");
             else
                 return QCoreApplication::translate("message-description sent:",
-                                                   "%1 sent some confetti")
+                                                   "%1 sent a special effect")
```

Comment:
Input would be appreciated here

Reply:
Maybe just "X sent a chat effect"? Or we do actually add some translations for a few known types, I wouldn't mind :)

Reply:
The main problem is that to do that I will have to do some refactoring to pass the original event into this function, but I suppose it can be done.

Reply:
Ah, yeah, that is a bit annoying. How about going with "sent a chat effect" for now and fixing it up later, so that Nheko master and mtxclient master are compatible again, while I refactor some other stuff?

Reply:
Sure